### ISSUE TITLE
Incorrect "missing trailing comma error" in method chain

### ISSUE THREAD

**Issue by olalonde** [eslint/eslint#8962]
I'm getting a missing trailing comma error at the `.use('/', userRouter)` line.

```javascript
app
    .use('/account', r()
      .use(isLoggedIn)
      .use((req, res, next) => {
        req.params.username = req.user.get('username')
        next()
      })
      .use('/', userRouter)
    )
```

Edit: ahh, is it because trailing comma is also required for function arguments split on more than one line?

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report.

> Edit: ahh, is it because trailing comma is also required for function arguments split on more than one line?

Yes.
